instrumentation/google.golang.org/grpc/otelgrpc/filters/interceptor/filters.go

```diff
@@ -0,0 +1,167 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package interceptor // import "go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc/filters/interceptor"
```

Comment:
Can you add a package documentation and mark the whole package as deprecated?

Reply:
thanks, I added package documentation and make it deprecated.
https://github.com/open-telemetry/opentelemetry-go-contrib/pull/5196/commits/c5175084cfae28588cc02c8e6bf9a778005ff163